pkg/apis/externaldns/validation/validation.go

```diff
@@ -45,16 +45,20 @@ func ValidateConfig(cfg *externaldns.Config) error {
 
 	// Akamai provider specific validations
 	if cfg.Provider == "akamai" {
-		if cfg.AkamaiServiceConsumerDomain == "" {
+		edgerc := false
+		if cfg.AkamaiEdgercPath != "" {
+			edgerc = true
+		}
+		if cfg.AkamaiServiceConsumerDomain == "" && !edgerc {
```

Comment:
would appreciate to keep it simple and not adding new vars

```suggestion
		if cfg.AkamaiServiceConsumerDomain == "" &&  cfg.AkamaiEdgercPath != ""  {
```

Reply:
Updated